### PR TITLE
Extend documentation by OCI OCIR

### PR DESCRIPTION
**Pull request by pgressa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,6 +280,33 @@ jobs:
...
+
+jobs:
+  login:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Login to OCIR
+        uses: docker/login-action@v1
+        with:
+          registry: <region-key>.ocir.io
```
Comment (crazy-max) [inline README.md:304]:
```suggestion
          registry: <region>.ocir.io
```

**Review by crazy-max — CHANGES_REQUESTED** [review]
Small typo

File: README.md (hunk 1 above)
Comment (pgressa) [reply README.md:304]: oh thanks for noticing!

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Pull request merged**